js/main.js

```diff
@@ -0,0 +1,113 @@
+var currentGyro = 0;
+var offsetGyro = 0;
+
+$(document).ready(function() {
+    $("#mainform").submit(function(e) {
+        e.preventDefault();
+        remove_form();
+    });
```

Comment:
Have you considered automatically hiding the checklist instead of making the drivers remember to press a button to hide it?